README.md

```
@@ -1,149 +1,150 @@
+[![OpenSSF Scorecard]( https://api.scorecard.dev/projects/github.com /hiero-ledger/hiero-sdk-python/badge)](https: //scorecard.dev/viewer/?uri=github.com/hiero- ledger/hiero-sdk-python)
+[![CII Best Practice s](https://bestpractices.coreinfrastructure.o rg/projects/10697/badge)](https://bestpractic es.coreinfrastructure.org/projects/10697)
+[![ License](https://img.shields.io/badge/license -apache2-blue.svg)](https://github.com/hiero- ledger/hiero-sdk-python/blob/main/LICENSE)
+[! [Python](https://img.shields.io/badge/python- 3.10%2B-blue)](https://www.python.org/)
 
-A Python SDK for interacting with the Hedera Hashgraph platform.
+A Py thon SDK for interacting with the Hedera Hash graph platform.
 
 >**Python compatibility:**
-> The SDK supports Python ≥ 3.10 and is tested on Python 3.10–3.14. Newer Python versions may work but have not yet been validated.
-
+>  The SDK supports Python ≥ 3.10 and is test ed on Python 3.10–3.14. Newer Python versio ns may work but have not yet been validated.
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)